JLToast/JLToastView.swift

```diff
@@ -26,6 +26,13 @@ public let JLToastViewTextColorAttributeName = "JLToastViewTextColorAttributeNam
 public let JLToastViewFontAttributeName = "JLToastViewFontAttributeName"
 public let JLToastViewPortraitOffsetYAttributeName = "JLToastViewPortraitOffsetYAttributeName"
 public let JLToastViewLandscapeOffsetYAttributeName = "JLToastViewLandscapeOffsetYAttributeName"
+public let JLToastViewGravityAttributeName = "JLToastViewGravityAttributeName"
+
+public struct JLToastGravity {
```

Comment:
I think we can define `JLToastGravity` with `enum`. Is this intended?
If you had considered Objective-C compatible, it should be defined in `JLToast.h`, too.